### PR TITLE
dashboard-create-nebula-ent&license

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-dashboard-ent/3.create-import-dashboard/2.import-cluster.md [hunk 1]
```diff
@@ -12,20 +12,28 @@ This topic introduces how to import clusters using Dashboard. The current versio
...
+   - Enter the cluster name, 15 characters at most. In this example, the cluster name is `create_1027`, and choose whether to use `sudo` to connect to the cluster.
+
+    !!! notice
+
+        If your SSH account does not have permission for the Nebula Graph cluster, you can use `sudo` to connect to it.
 
-3. In the **Connect to Nebula Graph** page, fill in the following:
-   - Enter the cluster name, 15 characters at most. In this example, the cluster name is `create_1027`.
-   - **Authorize** the node. The SSH username and password of each node are required.
+   - **Authorize** the node. The SSH username and password of each node are required, and choose
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/3.create-import-dashboard/2.import-cluster.md:36]: Unfinished sentence?

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md [hunk 2]
```diff
@@ -19,19 +21,35 @@ In this part, you can view the number of nodes as well as the number of running
 
 ### Alert
 
-In the **Alert** section, the system displays the five most recently triggered alert messages according to their severity level (`emergency`>`critical`>`warning`). For more information about alerts, see [Alerts](../9.alerts.md).
+In the **Alert** section, the system displays the five most recently triggered alert messages according to their severity level (`emergency`>`critical`>`warning`).
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md:24]:
```suggestion
In the **Alert** section, the system displays the five most recently triggered alert messages according to their severity levels (`emergency`>`critical`>`warning`). 
```

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md [hunk 3]
```diff
@@ -19,19 +21,35 @@ In this part, you can view the number of nodes as well as the number of running
...
-In this part, you can view the information of **Cluster name**, **Creation time**, **Creator**, and **Version**.
+In this part, you can view the information of **Cluster Name**, **Creation Time**, **Expiration Time**, **Creator**, and **Version**.
+
+- **Cluster Name**：The name of the cluster.
+- **Creation Time**：The time when the cluster is created.
+- **Expiration Time**：The time when the cluster license expires.
+
+  !!! enterpriseonly
+
+        The parameter **Expiration Time** is displayed only if the created or imported cluster is an Enterprise Edition.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md:38]: "Cluster is an Enterprise Edition"?

Consider:

The parameter **Expiration Time** is displayed only if the created or imported cluster is an Enterprise Edition cluster.
or
The parameter **Expiration Time** is displayed for only Enterprise Edition clusters.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md [hunk 4]
```diff
@@ -19,19 +21,35 @@ In this part, you can view the number of nodes as well as the number of running
...
+
+- **Creator**：The Dashboard account that used to create the cluster.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md:40]:
```suggestion
- **Creator**：The Dashboard account that is used to create the cluster. 
```

Or just "account used".

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md [hunk 5]
```diff
@@ -19,19 +21,35 @@ In this part, you can view the number of nodes as well as the number of running
...
+- **Expiration Time**：The time when the cluster license expires.
+
+  !!! enterpriseonly
+
+        The parameter **Expiration Time** is displayed only if the created or imported cluster is an Enterprise Edition.
+
+- **Creator**：The Dashboard account that used to create the cluster. 
+- **Version**：The version of Nebula Graph installed in the cluster.
+
+In the upper right of the **Information** section, click ![watch](https://docs-cdn.nebula-graph.com.cn/figures/watch.png) to view the cluster details, including name, creation time, account name, version, the role of the account name.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md:43]:
```suggestion
In the upper right of the **Information** section, click ![watch](https://docs-cdn.nebula-graph.com.cn/figures/watch.png) to view the cluster details, including name, creation time, account name, version, and the role of the account name.
```

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md [hunk 6]
```diff
@@ -19,19 +21,35 @@ In this part, you can view the number of nodes as well as the number of running
...
+
+!!! enterpriseonly
 
-!!! note
+    For Enterprise Edition, there is a **License** section.
 
-    The version here is the Nebula Graph version installed by the user, not the Dashboard version.
+    - Displays the license details of the cluster, including the usage status, the organization, the creation time, the expiration time, and the cluster versions supported by the license, and the license type.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md:49]:
```suggestion
    - Displays the license details of the cluster, including the usage status, the organization, the creation time, the expiration time, the cluster versions supported by the license, and the license type.
```

**Review by randomJoe211 — CHANGES_REQUESTED** [review]
Complete the unfinished sentence, please.

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**